lib/tail-file.js

```diff
@@ -68,6 +69,24 @@ class TailFile extends Readable {
       }
       throw err
     }
+    if (startPos !== null && startPos !== undefined) {
+      if (typeof startPos !== 'number') {
+        const err = new TypeError('startPos must be an integer >= 0')
+        err.code = 'ESTARTPOS'
+        err.meta = {
+          got: typeof startPos
+        }
+        throw err
+      }
+      if (startPos < 0 || !Number.isInteger(startPos)) {
+        const err = new TypeError('startPos must be an integer >= 0')
```

Comment:
`RangeError` or just `Error`? I think `RangeError` is appropriate for when a positive non-integer value is provided. But doesn't seem like this should be `TypeError`.

> The RangeError object indicates an error when a value is **not in the set** or range of allowed values.

Reply:
yep, that works